static/app/views/issueList/overview.tsx

```diff
@@ -607,10 +603,7 @@ class IssueListOverview extends Component<Props, State> {
           return;
         }
 
-        if (this.state.undo) {
```

Comment:
I couldn't figure out why this is here, when the next line pushed the data anyway?